### ISSUE TITLE
training configuration

### ISSUE THREAD

**Issue by zhangxuan1918** [tranluan/Nonlinear_Face_3DMM#26]
Hello, 

I tried to train the model. However I got a few issues. During pre-training, after 200 epochs, the training converges and doesn't improve. The rendered image doesn't look good, especially the pose and texture. Then I switched to unsupervised training. However after 100 epochs, the image rendered is super blurred. 

I'm not sure if the configuration in the code is the default setting you use. Thus, could you share your configuration for training, e.g.

pre-training:

* number of epochs

training
* number of epochs, in code, it says 1000, do you really use 1000 epochs?
* which loss you use? Do you use all the loss mentioned in the paper/code?

Thanks a lot

**Comment by linshuheng6** [comment]
Hello, I have no idea about how to run the pre_training model. Could you please share your configuration for pre_training. Thank you !!!

**Comment by HOMGH** [comment]
> Hello,
> 
> I tried to train the model. However I got a few issues. During pre-training, after 200 epochs, the training converges and doesn't improve. The rendered image doesn't look good, especially the pose and texture. Then I switched to unsupervised training. However after 100 epochs, the image rendered is super blurred.
> 
> I'm not sure if the configuration in the code is the default setting you use. Thus, could you share your configuration for training, e.g.
> 
> pre-training:
> 
> * number of epochs
> 
> training
> 
> * number of epochs, in code, it says 1000, do you really use 1000 epochs?
> * which loss you use? Do you use all the loss mentioned in the paper/code?
> 
> Thanks a lot

Hi @zhangxuan1918 
I'm trained the model. but the results (rendered images and textures) are not reasonable. I used just 1000 images and 40 epochs for training. Could you train the model correctly?
